package.json

```diff
@@ -79,9 +79,11 @@
     "eslint-plugin-node": "^11.1.0",
     "eslint-plugin-standard": "^5.0.0",
     "eslint-plugin-vue": "^7.9.0",
+    "express-ws": "^4.0.0",
```

Comment:
`json-server` uses Express, but for WebSockets we need `express-ws` as the built-in `ws` module doesn't seem to work well with Express :man_shrugging: https://stackoverflow.com/a/32705838